syntax = "proto3";

message RequestMsg
{
    int32 cmdType = 1;
    bytes clientId = 2;
    bytes serverId = 3;
    bytes sign = 4;
    bytes data = 5;
}

message RespondMsg
{
    bool rv = 1;
    bytes clientId = 2;
    bytes serverId = 3;
    int32 seckeyid = 4;
    bytes data = 5;
}